Traceback (most recent call last):
  File "/home/pi/longibot-raspberry/web/arduino-cmds/py/start_serial.py", line 3, in <module>
    from logs import iflan_log
  File "/home/pi/longibot-raspberry/web/arduino-cmds/py/logs.py", line 7, in <module>
    logging.basicConfig(filename=LOG, filemode="w", level=logging.DEBUG)  
  File "/usr/lib/python3.7/logging/__init__.py", line 1900, in basicConfig
    h = FileHandler(filename, mode)
  File "/usr/lib/python3.7/logging/__init__.py", line 1092, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.7/logging/__init__.py", line 1121, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/logs/iflan.log'
Traceback (most recent call last):
  File "/home/pi/longibot-raspberry/web/arduino-cmds/py/start_serial.py", line 3, in <module>
    from logs import iflan_log
  File "/home/pi/longibot-raspberry/web/arduino-cmds/py/logs.py", line 7, in <module>
    logging.basicConfig(filename=LOG, filemode="w", level=logging.DEBUG)  
  File "/usr/lib/python3.7/logging/__init__.py", line 1900, in basicConfig
    h = FileHandler(filename, mode)
  File "/usr/lib/python3.7/logging/__init__.py", line 1092, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.7/logging/__init__.py", line 1121, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/logs/iflan.log'
Traceback (most recent call last):
  File "/home/pi/longibot-raspberry/web/arduino-cmds/py/start_serial.py", line 3, in <module>
    from logs import iflan_log
  File "/home/pi/longibot-raspberry/web/arduino-cmds/py/logs.py", line 7, in <module>
    logging.basicConfig(filename=LOG, filemode="w", level=logging.DEBUG)  
  File "/usr/lib/python3.7/logging/__init__.py", line 1900, in basicConfig
    h = FileHandler(filename, mode)
  File "/usr/lib/python3.7/logging/__init__.py", line 1092, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.7/logging/__init__.py", line 1121, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/logs/iflan.log'
Traceback (most recent call last):
  File "/home/pi/longibot-raspberry/web/arduino-cmds/py/start_serial.py", line 3, in <module>
    from logs import iflan_log
  File "/home/pi/longibot-raspberry/web/arduino-cmds/py/logs.py", line 7, in <module>
    logging.basicConfig(filename=LOG, filemode="w", level=logging.DEBUG)  
  File "/usr/lib/python3.7/logging/__init__.py", line 1900, in basicConfig
    h = FileHandler(filename, mode)
  File "/usr/lib/python3.7/logging/__init__.py", line 1092, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.7/logging/__init__.py", line 1121, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/logs/iflan.log'
Traceback (most recent call last):
  File "/home/pi/longibot-raspberry/web/arduino-cmds/py/start_serial.py", line 3, in <module>
    from logs import iflan_log
  File "/home/pi/longibot-raspberry/web/arduino-cmds/py/logs.py", line 7, in <module>
    logging.basicConfig(filename=LOG, filemode="w", level=logging.DEBUG)  
  File "/usr/lib/python3.7/logging/__init__.py", line 1900, in basicConfig
    h = FileHandler(filename, mode)
  File "/usr/lib/python3.7/logging/__init__.py", line 1092, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.7/logging/__init__.py", line 1121, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/logs/iflan.log'
Traceback (most recent call last):
  File "/home/pi/longibot-raspberry/web/arduino-cmds/py/start_serial.py", line 3, in <module>
    from logs import iflan_log
  File "/home/pi/longibot-raspberry/web/arduino-cmds/py/logs.py", line 7, in <module>
    logging.basicConfig(filename=LOG, filemode="w", level=logging.DEBUG)  
  File "/usr/lib/python3.7/logging/__init__.py", line 1900, in basicConfig
    h = FileHandler(filename, mode)
  File "/usr/lib/python3.7/logging/__init__.py", line 1092, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.7/logging/__init__.py", line 1121, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/logs/iflan.log'
Traceback (most recent call last):
  File "/home/pi/longibot-raspberry/web/arduino-cmds/py/start_serial.py", line 3, in <module>
    from logs import iflan_log
  File "/home/pi/longibot-raspberry/web/arduino-cmds/py/logs.py", line 7, in <module>
    logging.basicConfig(filename=LOG, filemode="w", level=logging.DEBUG)  
  File "/usr/lib/python3.7/logging/__init__.py", line 1900, in basicConfig
    h = FileHandler(filename, mode)
  File "/usr/lib/python3.7/logging/__init__.py", line 1092, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.7/logging/__init__.py", line 1121, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/logs/iflan.log'
Traceback (most recent call last):
  File "/home/pi/longibot-raspberry/web/arduino-cmds/py/start_serial.py", line 3, in <module>
    from logs import iflan_log
  File "/home/pi/longibot-raspberry/web/arduino-cmds/py/logs.py", line 7, in <module>
    logging.basicConfig(filename=LOG, filemode="w", level=logging.DEBUG)  
  File "/usr/lib/python3.7/logging/__init__.py", line 1900, in basicConfig
    h = FileHandler(filename, mode)
  File "/usr/lib/python3.7/logging/__init__.py", line 1092, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.7/logging/__init__.py", line 1121, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/logs/iflan.log'
Traceback (most recent call last):
  File "/home/pi/longibot-raspberry/web/arduino-cmds/py/start_serial.py", line 3, in <module>
    from logs import iflan_log
  File "/home/pi/longibot-raspberry/web/arduino-cmds/py/logs.py", line 7, in <module>
    logging.basicConfig(filename=LOG, filemode="w", level=logging.DEBUG)  
  File "/usr/lib/python3.7/logging/__init__.py", line 1900, in basicConfig
    h = FileHandler(filename, mode)
  File "/usr/lib/python3.7/logging/__init__.py", line 1092, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.7/logging/__init__.py", line 1121, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/logs/iflan.log'
Traceback (most recent call last):
  File "/home/pi/longibot-raspberry/web/arduino-cmds/py/start_serial.py", line 3, in <module>
    from logs import iflan_log
  File "/home/pi/longibot-raspberry/web/arduino-cmds/py/logs.py", line 7, in <module>
    logging.basicConfig(filename=LOG, filemode="w", level=logging.DEBUG)  
  File "/usr/lib/python3.7/logging/__init__.py", line 1900, in basicConfig
    h = FileHandler(filename, mode)
  File "/usr/lib/python3.7/logging/__init__.py", line 1092, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.7/logging/__init__.py", line 1121, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/logs/iflan.log'
Traceback (most recent call last):
  File "/home/pi/longibot-raspberry/web/arduino-cmds/py/start_serial.py", line 3, in <module>
    from logs import iflan_log
  File "/home/pi/longibot-raspberry/web/arduino-cmds/py/logs.py", line 7, in <module>
    logging.basicConfig(filename=LOG, filemode="w", level=logging.DEBUG)  
  File "/usr/lib/python3.7/logging/__init__.py", line 1900, in basicConfig
    h = FileHandler(filename, mode)
  File "/usr/lib/python3.7/logging/__init__.py", line 1092, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.7/logging/__init__.py", line 1121, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/logs/iflan.log'
Traceback (most recent call last):
  File "/home/pi/longibot-raspberry/web/arduino-cmds/py/start_serial.py", line 3, in <module>
    from logs import iflan_log
  File "/home/pi/longibot-raspberry/web/arduino-cmds/py/logs.py", line 7, in <module>
    logging.basicConfig(filename=LOG, filemode="w", level=logging.DEBUG)  
  File "/usr/lib/python3.7/logging/__init__.py", line 1900, in basicConfig
    h = FileHandler(filename, mode)
  File "/usr/lib/python3.7/logging/__init__.py", line 1092, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.7/logging/__init__.py", line 1121, in _open
    return open(self.baseFilename, self.mode, encoding=self.encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/logs/iflan.log'
